docs/manufacturer/cloud-connector/activator/index.md

```diff
@@ -1,6 +1,8 @@
 # tapio Activator
 
-tapio Activator is a piece of software, which we're using to onboard your CloudConnector ( the physical world of woodworking machinery), with our digital ecosystem. It gets installed on the machine itself, or on a computer, which has network access to the machine.
+tapio Activator is a piece of software, which we're using to onboard your CloudConnector (the physical world of woodworking machinery), with our digital ecosystem. It gets installed on the machine itself, or on a computer, which has network access to the machine.
+
+> Note: the Activator was replaced by the browser based [CloudConnector UI](../Ui/). It is only available in older versions of the CloudConnector.
```

Comment:
This information should be more present imho. I suggest you use one of [admonitions](https://docusaurus.io/docs/next/markdown-features/admonitions) from docusaurus.